modules/install/pages/upgrade.adoc

```diff
@@ -158,6 +147,19 @@ If you're interested in upgrading to Couchbase Server Enterprise Edition, check
 See xref:install:upgrade-procedure-selection.adoc[Upgrade Procedure-Selection], for a list of procedures that can be used when upgrading from Community Edition to Enterprise.
 Note, however, that _Graceful Failover_ for Data Service nodes, with _Delta Recovery_, is _not_ supported for such upgrades: instead, _removal_, _addition_, and _swap rebalance_ should be used; for all nodes.
 
+[#node-certificate-validation]
+== Node Certificate Validation
+
+In Couchbase Enterprise Server Version 7.2+, the node-name _must_ be correctly identified in the node certificate as a Subject Alternative Name.
+If such identification is not correctly configured, failure may occur when uploading the certificate, or when attempting to add or join the node to a cluster.
+
+When upgrade is performed from a pre-7.2 cluster to a 7.2+ cluster, if a new, 7.2 node is introduced to the cluster without the node-name being correctly configured in its certificate, a warning is provided.
```

Comment:
@BahadorBiglar, can you please review this piece? It doesn't seem correct to me but I am not sure.